derive/src/pyclass.rs

```diff
@@ -206,6 +250,19 @@ pub(crate) fn impl_pystruct_sequence(
         }
     }
 
+    let (with_extend, with_slots) = if with_pyimpl {
```

Comment:
Will adding a new trait remove this conditional code? Probably `PyStructSequenceImpl` may include empty `__extend_py_class` just like `PyClassImpl` does